docs/configuration.rst

```diff
@@ -67,6 +67,14 @@ Toolbar options
   This setting is a set of the full Python paths to each panel that you
   want disabled (but still displayed) by default.
 
+* ``EDITOR_URL_CALLBACK``
+
+  Default: ``lambda f: None``
+
+  This function is passed a file path and should return a URL to open the file in
+  the editor. For example, to open the file in VS Code, you can use:
+  ``lambda f: f"vscode://file/{f}"``
```

Comment:
Is this ever not going to be a static prefix with the filename after it? I think if we can remove the lambda, that would be better.